src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -6,10 +6,12 @@
 public class CliSyntax {
 
     /* Prefix definitions */
+    public static final Prefix PREFIX_PID = new Prefix("id/");
     public static final Prefix PREFIX_NAME = new Prefix("n/");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
-    public static final Prefix PREFIX_EMAIL = new Prefix("e/");
-    public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
+    public static final Prefix PREFIX_PRENAME = new Prefix("p/");
+    public static final Prefix PREFIX_FOOD = new Prefix("f/");
```

Comment:
Similar to @SherisseTJW 's comment, maybe can spell out to be PREFIX_FOOD_PREFERENCE instead to deter confusion with other food related information such as FOOD_ALLERGIES.

Reply:
Have been updated to `PREFIX_FOOD_PREFERENCE`